engines/bastion_katello/app/assets/javascripts/bastion_katello/content-hosts/content/views/content-host-packages-installed.html

```diff
@@ -2,7 +2,7 @@
 
 <h3 translate>Installed Packages</h3>
 
-<div data-extend-template="common/views/katello-agent-notice.html"></div>
+<div ng-show="host.content_facet_attributes.katello_agent_installed" data-extend-template="common/views/katello-agent-notice.html"></div>
```

Comment:
Why make some of these display based on presence of katello-agent package vs the enablement of the agent functionality on the server itself?

Reply:
right, I guess that's the same thing I am asking with the table above!

Reply:
I will update it and change it, I saw that we could display stuff based on that, rather than adding new stuff into the controller I figured we could use that in the view. I will switch it the way we discussed.